AnkiDroid/src/main/java/com/ichi2/anki/Info.java

```diff
@@ -218,9 +218,13 @@ public String copyDebugInfo() {
             Timber.w(e, "Unable to detect Rust Backend");
         }
 
-
+        String webviewUserAgent = getWebviewUserAgent();
         String debugInfo = "AnkiDroid Version = " + VersionUtils.getPkgVersionName() + "\n\n" +
                 "Android Version = " + Build.VERSION.RELEASE + "\n\n" +
+                "Manufacturer = " + Build.MANUFACTURER + "\n\n" +
+                "Model = " + Build.MODEL + "\n\n" +
+                "Hardware = " + Build.HARDWARE+ "\n\n" +
```

Comment:
And we're done after this, thanks for the patience

```suggestion
                "Hardware = " + Build.HARDWARE + "\n\n" +
```